app/main.js

```diff
@@ -43,7 +45,10 @@ global.shared = {
 const gotTheLock = app.requestSingleInstanceLock()
 
 if (!gotTheLock) {
-  console.log('Stretchly is already running.')
+  console.log('Stretchly command instance: started\n')
```

Comment:
Only wondering: why do you use `\n`? :)

Reply:
Because I find not quite redeable when help is printed : 

```
Stretchly command instance: started
Usage: stretchly <command> [options]

Commands:
	stretchly help            Show this help message
	stretchly version         Show current stretchly version
	stretchly reset           Reset breaks
	stretchly pause [options] Pause breaks
	stretchly resume          Resume from a pause
	stretchly toggle          Toggle breaks between resume/paused
	stretchly mini [options]  Skips to next Mini Break
	stretchly long [options]  Skips to next Long Break

Options:
	-T, --title    Specify title for next break (Long or Mini)
	-t, --text     Specify text for next break (Long Break only)
	-s, --skip     Skip directly to this break (Long or Mini)
	-d, --duration Specify duration for pausing breaks (Pause only) [infinitely|until-morning|seconds (as number)]

Examples:
	stretchly pause                                    Pause breaks indefinitly
	stretchly pause -d 3600                            Pause breaks for one hour
	stretchly mini -T "Stretch up !" --skip            Skips to next Mini Break with "Stretch up!" title
	stretchly long -T "Stretch up !" -t "Go stretch !" Set next long break title to "Stretch up !" and text to "Go stretch !"
```

vs

```
Stretchly command instance: started
Usage: stretchly <command> [options]

Commands:
	stretchly help            Show this help message
	stretchly version         Show current stretchly version
	stretchly reset           Reset breaks
	stretchly pause [options] Pause breaks
	stretchly resume          Resume from a pause
	stretchly toggle          Toggle breaks between resume/paused
	stretchly mini [options]  Skips to next Mini Break
	stretchly long [options]  Skips to next Long Break

Options:
	-T, --title    Specify title for next break (Long or Mini)
	-t, --text     Specify text for next break (Long Break only)
	-s, --skip     Skip directly to this break (Long or Mini)
	-d, --duration Specify duration for pausing breaks (Pause only) [infinitely|until-morning|seconds (as number)]

Examples:
	stretchly pause                                    Pause breaks indefinitly
	stretchly pause -d 3600                            Pause breaks for one hour
	stretchly mini -T "Stretch up !" --skip            Skips to next Mini Break with "Stretch up!" title
	stretchly long -T "Stretch up !" -t "Go stretch !" Set next long break title to "Stretch up !" and text to "Go stretch !"
```

But it's a matter of taste

Reply:
Ah, ok, you're right :)